workshop/content/basics/templates/finding-return-values/_index.md

```diff
@@ -0,0 +1,181 @@
+---
```

Comment:
@dreddy-aws can you actually rename the directory to `resource-return-values` for consistency with the code directories?  You might also need to change the image path in the markdown file.  Thanks!